### PR TITLE
fix uc events

### PR DESCRIPTION
**Pull request by tmsdkeys**
This PR:

- fixes packet event filtering for universal channels (event shows UC handler address, not app)
- some additional cleanup related to #31 

## Summary by CodeRabbit

- **New Features**
	- Added functions for retrieving specific addresses based on network type, enhancing script flexibility and adaptability.
- **Refactor**
	- Updated logic for determining dispatcher and universal channel handler addresses, improving address retrieval based on network conditions.

### CODE REVIEW

File: scripts/_events.js [hunk 1]
```diff
@@ -114,12 +115,15 @@ function listenForIbcChannelEvents(network, source, dispatcher) {
 
 function filterPacketEvents(portAddress, network) {
     const sendPacketConfig = config.sendPacket;
-    const sendUniversalPacketConfig = config.sendUniversalPacket;
-
-    const filterCondition = 
-        portAddress === sendPacketConfig[network].portAddr ||
-        portAddress === sendUniversalPacketConfig[network].portAddr;
-    return filterCondition;
+    let ucHandlerAddr;
```
Comment (kenobijon) [inline scripts/_events.js:118]: Recommend to use getUcHandler from `_get_vibc_src.js` to ensure standard patterns

**Comment by kenobijon** [comment]
When testing, the sending packets over UC is working but the script is stalling out and not logging any of the events event after a successful packet lifecycle:

```
❯ just send-packet optimism true
Attempting to send a packet over the universal channel as defined in the config...
👂 Listening for IBC packet events on optimism...
👂 Listening for IBC packet events on base...
Fetching IBC app on optimism at address: 0xBeA3514300C422792195e0E9b187189E2AfEde03
```

**Review by kenobijon — CHANGES_REQUESTED** [review]

File: scripts/_events.js (hunk 1 above)
Comment (tmsdkeys) [reply scripts/_events.js:118]: So getUcHandler fetches the contract itself, not just the address. But I do see your point and could maybe add a separate helper function getUcHandlerAddr instead?

**Review by kenobijon — APPROVED** [review]
Looks good to me!

**Pull request merged**